Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4394,6 +4465,357 @@ def arg_to_datetime(arg, arg_name=None, is_utc=True, required=False, settings=No
+                 relation_type='indicatorToIndicator', entity_a_family='Indicator', entity_b_family='Indicator',
+                 source_reliability="", fields=None, brand=""):
+
+        # Relation
+        if not Relations.is_valid(name):
+            raise ValueError("Invalid relation: " + name)
+        self._name = str(name)
+
+        self._reverse_name = Relations.get_reverse(str(name))
+
+        if not RelationsTypes.is_valid_type(relation_type):
+            raise ValueError("Invalid relation type: " + relation_type)
+        self._relation_type = str(relation_type)
+
+        # Entity A - Source
+        self._entity_a = str(entity_a)
+
+        # The type of the entity doesnt have to be a indicator type
+        if not FeedIndicatorType.is_valid_type(object_type_a):
+            raise ValueError("Invalid entity A type: " + object_type_a)
+        self._object_type_a = str(object_type_a)
+
+        if not RelationsFamily.is_valid_type(entity_a_family):
+            raise ValueError("Invalid entity A Family type: " + entity_a_family)
+        self._entity_a_family = str(entity_a_family)
+
+        # Entity B - Destination
+        self._entity_b = str(entity_b)
+
+        # The type of the entity doesnt have to be a indicator type
+        if not FeedIndicatorType.is_valid_type(object_type_b):
+            raise ValueError('Invalid entity B type: ' + object_type_b)
+        self._object_type_b = str(object_type_b)
+
+        if not RelationsFamily.is_valid_type(entity_b_family):
+            raise ValueError("Invalid entity B Family type: " + entity_b_family)
+        self._entity_b_family = str(entity_b_family)
+
+        # Custom fields
+        if fields:
+            self._fields = fields
+        else:
+            self._fields = {}
+
+        # Source
+        if brand:
+            self._brand = brand
+        else:
+            self._brand = ''
+
+        if source_reliability:
+            if not DBotScoreReliability.is_valid_type(source_reliability):
+                raise ValueError("Invalid source reliability value", source_reliability)
+            self._source_reliability = source_reliability
+
+    def to_entry(self):
+        """ Convert object to XSOAR entry
+        :rtype: ``dict``
+        :return: XSOAR entry representation.
+        """
+        entry = {
+            "name": self._name,
+            "reverseName": self._reverse_name,
+            "type": self._relation_type,
+            "entityA": self._entity_a,
+            "entityAFamily": self._entity_a_family,
+            "objectTypeA": self._object_type_a,
+            "entityB": self._entity_b,
+            "entityBFamily": self._entity_b_family,
+            "objectTypeB": self._object_type_b,
+            "fields": self._fields,
+            "reliability": self._source_reliability
+        }
+        if self._brand:
+            entry["brand"] = self._brand
+        return entry
+
+    def to_indicator(self):
```

Comment:
I think that indicator has a reliability and a brand.

Reply:
also, when we would use that? I thought that relationships can be returned only with indicators.

Reply:
Relationships can be returned from feeds which will return as part of the indicator and also as part of the enrichment commands.
There is a small difference between the object needed from feed and from enrichments.
For example, there is no need for a brand in feeds, as server set it. Also the same for Reliability